src/components/ContactsTable.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useState } from 'react';
+import contactsJson from '../contacts.json';
+import Contact from './Contact';
+
+const filteredContacts = contactsJson.map(contact => contact);
+
+for (let i = 0; i < 5; i++) {
```

Comment:
Este for tampoco te haria falta, ya que lo que haces despues es lo del slice que esta genial